evalml/pipelines/components/estimators/classifiers/lightgbm_classifier.py

```diff
@@ -30,7 +30,7 @@ class LightGBMClassifier(Estimator):
     SEED_MIN = 0
     SEED_MAX = SEED_BOUNDS.max_bound
 
-    def __init__(self, boosting_type="gbdt", learning_rate=0.1, n_estimators=100, max_depth=0, num_leaves=31, min_child_samples=20, n_jobs=-1, random_state=0, **kwargs):
+    def __init__(self, boosting_type="gbdt", learning_rate=0.1, n_estimators=100, max_depth=0, num_leaves=31, min_child_samples=20, n_jobs=-1, random_state=0, bagging_fraction=0.9, bagging_freq=0, **kwargs):
```

Comment:
Why default `bagging_freq` to 0? Won't that cause the bug when `boosting_type="rf"`? What default does lightgbm choose for this parameter?

Reply:
LightGBM defaults to 0 for `bagging_freq`. Users can set it to 1 and change `bagging_fraction` if they want to speed up computation and randomly select data for other boosting types, but it's required to be 1 for `boosting_type=rf` (along with `0 < bagging_fraction < 1.0`).

Reply:
Got it. This looks good. Is 0.9 the default `bagging_fraction` in lightgbm?

Reply:
@dsherry it defaults to 1.0